contracts/evoting/controller/action.go

```diff
@@ -44,7 +44,15 @@ import (
 )
 
 const contentType = "application/json"
+const electionPath = "/evoting/elections"
+const electionPathSlash = "/evoting/elections/"
+const electionIDPath = "/evoting/elections/{electionID}"
```

Comment:
Since those three constants are interdependable you might want to define the last two according to the first one, or not define the last two at all. This makes it easier to refactor later on